hunts-api/routes/alive.js

```diff
@@ -3,6 +3,22 @@ const getDb = require('../db');
 const definition = require('../package.json');
 
 
+<<<<<<< HEAD
```

Comment:
@esayers85 These conflict markers mean you had conflicts that you did not resolve. I also don't think that you need changes in this file  at all, so perhaps replace the whole thing with what is on my repo